components/resources/strings/brave_components_resources_en-GB.xtb

```diff
@@ -376,7 +376,7 @@
 <translation id="2070991379950719711">Start using Brave Rewards</translation>
 <translation id="6318696841371191107">Take a quick tour</translation>
 <translation id="8465790416279415448">By proceeding, you agree to the <ph name="TOS_LINK_BEGIN"/>Terms of Service<ph name="TOS_LINK_END"/> and <ph name="PP_LINK_BEGIN"/>Privacy Policy<ph name="PP_LINK_END"/>.</translation>
-<translation id="2404165674840145940">Tip with Brave rewards</translation>
+<translation id="2404165674840145940">Tip with Brave Rewards</translation>
```

Comment:
No need to modify this file; it's auto-generated and then uploaded to our translators.

Reply:
Should I revert that change?

Reply:
Yeah, I think that makes sense. The translators will update all of the .XTB's in one go the next time we order translations.

Reply:
Consider it done :D